images/golang-builder/Dockerfile

```diff
@@ -100,17 +100,17 @@ RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 # Install golangci-lint for linting
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.22.2
 # Install ginkgo for running tests
-RUN go get github.com/onsi/ginkgo/ginkgo
+RUN GO111MODULE=on go get github.com/onsi/ginkgo/ginkgo@v1.14.0
 
 # Install more lint and code generation tools
-RUN go get golang.org/x/tools/cmd/goimports \
-    && go get github.com/kisielk/errcheck \
-    && go get github.com/gordonklaus/ineffassign \
-    && go get github.com/fzipp/gocyclo \
-    && go get honnef.co/go/tools/... \
-    && go get golang.org/x/lint/golint \
-    && go get gopkg.in/golang/mock.v1/gomock \
-    && go get gopkg.in/golang/mock.v1/mockgen
+RUN go get golang.org/x/tools/cmd/goimports
```

Comment:
Having multiple `RUN`s would slow down the build, although I am not sure if that is a problem in this case.

Reply:
I leave it to to your judgement.

Reply:
Ill update it now, I set it like this to make it easier to debug and didn't switch back 😅